README.md

```diff
@@ -91,6 +92,79 @@ If you want to change how long each test will be run, you can change the times w
 pgbench_command: pgbench -c 32 -j 16 -T <test time in seconds> -P 10 -r
 ```
 
+## <a name="running-automated-hammerdb"></a>Running Automated Hammerdb
+
+Hammerdb tests are run from a driver node. Driver node is in the same virtual network as the cluster.
+You can customize the hammerdb cluster in the `hammerdb` folder using `azuredeploy.parameters.json`.
+
+**You should create a new branch and change the settings in the new branch and push the branch so that
+when the tool clones the repository it can download your branch.**
+
+In order to run hammerdb benchmark:
+
+```bash
+eval `ssh-agent -s`
+ssh-add
+EXPORT RESOURCE_GROUP_NAME=<your resource group name>
+EXPORT GIT_USERNAME=<Your github username>
+EXPORT GIT_TOKEN=<Your github token with at least write and read access> # You can create a github token from https://github.com/settings/tokens. 
```

Comment:
export should be lower case

Reply:
I am thinking of running a shell check here and updating the style accordingly for shell scripts.